
Primarily needed for running the suite with python 3.5, since the xattrs returned
with that version on Solaris are reversed to previous versions. Not contributed
upstream, but may be in future.

--- xattr-0.9.3/xattr/tests/test_xattr.py.orig	2018-02-26 20:31:20.585546960 -0800
+++ xattr-0.9.3/xattr/tests/test_xattr.py	2018-02-26 20:31:31.999288779 -0800
@@ -26,11 +26,13 @@
 
         # SELinux systems use an attribute which must be accounted for
         if sys.platform.startswith('linux') and 'security.selinux' in x:
             d['security.selinux'] = x['security.selinux']
 
-        self.assertEqual(list(x.keys()), list(d.keys()))
-        self.assertEqual(list(x.list()), list(d.keys()))
+        listx = list(x.keys()).sort()
+        listd = list(d.keys()).sort()
+        self.assertEqual(listx, listd)
+        self.assertEqual(list(x.list()).sort(), listd)
         self.assertEqual(dict(x), d)
 
         x['user.sopal'] = b'foo'
         x['user.sop.foo'] = b'bar'
